src/main/java/uk/gov/hmcts/probate/validator/SolicitorPostcodeValidationRule.java

```diff
@@ -36,7 +37,7 @@ public List<FieldErrorResponse> validate(CCDData ccdData) {
     private List<String> getErrorCodeForSolicitorPostCodeMissing(CCDData ccdData) {
         List<String> allErrorCodes = new ArrayList<>();
         SolsAddress solicitorAddress = ccdData.getSolicitor().getFirmAddress();
-        if (StringUtils.isEmpty(solicitorAddress.getPostCode())) {
+        if (ccdData.getApplicationType().equals(SOLICITOR.toString()) && StringUtils.isEmpty(solicitorAddress.getPostCode())) {
```

Comment:
its better practice to have the 
SOLICITOR.toString().equals(ccdData.getApplicationType())